### PR TITLE
Feature/login backend dev

### PR DESCRIPTION
**Pull request by pacmadhushan**
complete login api

### CODE REVIEW

File: src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/CertificateAPI.cs [hunk 1]
```diff
@@ -28,9 +28,7 @@ public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
             var certificate = await System.Text.Json.JsonSerializer.DeserializeAsync<Certificate>(req.Body);
 
             var response = req.CreateResponse(HttpStatusCode.OK);
-            response.Headers.Add("Content-Type", "text/plain; charset=utf-8");
-
-
+          
             var created = await repository.CreateAsync(new Certificate { Course = certificate.Course, Trainee = certificate.Trainee, Trainer = certificate.Trainer, CertificateIssueDate =certificate.CertificateIssueDate });
```
Comment (chan4lk) [inline src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/CertificateAPI.cs:32]: This line can be simply written as `var created = await repository.CreateAsync(certificate);`

File: src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TraineeAPI.cs [hunk 2]
```diff
@@ -32,10 +32,7 @@ public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
 
             var created =await repository.CreateAsync(new Trainee { FirstName = trainee.FirstName, LasttName = trainee.LasttName, EmailAddress = trainee.EmailAddress });
```
Comment (chan4lk) [inline src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TraineeAPI.cs:33]: This line can be simply written as `var created = await repository.CreateAsync(trainee);`

File: src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TrainerAPI.cs [hunk 3]
```diff
@@ -32,8 +32,6 @@ public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymou
             var created = await repository.CreateAsync(new Trainer { FirstName = trainer.FirstName, LasttName = trainer.LasttName, EmailAddress = trainer.EmailAddress });
```
Comment (chan4lk) [inline src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TrainerAPI.cs:32]: This line can be simply written as `var created = await repository.CreateAsync(trainer);`

**Review by chan4lk — COMMENTED** [review]
Please simplify the statements

File: src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/LoginAPI.cs [hunk 4]
```diff
@@ -23,15 +25,22 @@ public async Task<HttpResponseData> RunAsync([HttpTrigger(AuthorizationLevel.Ano
...
-
+             var  users = await repository.GetAsync(x => x.UserName == temp.UserName && x.Password == temp.Password);
+
+            if (users.Count()==1)
+            {
+                response.WriteString("Welcome to Bistec Global!");
+            }
+            else
+            {
+                 response = req.CreateResponse(HttpStatusCode.Unauthorized);
```
[the remark this answers is not in the document]
Comment (pacmadhushan) [reply src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/LoginAPI.cs]: added logger

File: src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TrainerAPI.cs (hunk 3 above)
Comment (pacmadhushan) [reply src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TrainerAPI.cs]: Done

File: src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TraineeAPI.cs (hunk 2 above)
Comment (pacmadhushan) [reply src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/TraineeAPI.cs]: Done

File: src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/CertificateAPI.cs (hunk 1 above)
Comment (pacmadhushan) [reply src/backend/BlockchainCertificatesIssuer/BlockchainCertificatesIssuer/API/CertificateAPI.cs]: Done

**Review by dulankadam — APPROVED** [review]

**Pull request merged**